tests/testthat/test-tabPanel.R

```diff
@@ -18,6 +18,9 @@ expect_snapshot2 <- function(...) {
   if (getRversion() < "3.6.0") {
     skip("Skipping snapshots on R < 3.6 because of different RNG method")
   }
+  if (packageVersion("htmltools") <= "0.5.6") {
+    skip("Skipping snapshots since htmltools is 'outdated'")
+  }
```

Comment:
This is needed since these `tabPanel()` tests ultimately call `tagQuery()` in such a way that https://github.com/rstudio/htmltools/pull/408 becomes problematic (for r-devel).